### PR TITLE
force level if not working

### PR DESCRIPTION
**Pull request by kay0u**
If an app have the state "notworking", then force the level to 0.

Should I do the same for "inprogress"?

### CODE REVIEW

**Comment by kay0u** [comment]
Bump

**Comment by alexAubin** [comment]
Uuuuh can you confirm that it's what's actually used to update app levels ?

Code-wise, to me it should be more something like : 

`if app["state"] != "working":
    del app["level"]`

so that it entirely deletes the level key compared to setting it to 0 (which suggests a test was made) but that's kind of a detail, dunno if we really care so much about it, the important bit being that it's not a value higher than 0...

**Comment by kay0u** [comment]
https://github.com/YunoHost/CI_package_check/blob/594a25480a7af39bf8cf9b853955df267d73a857/auto_build/update_level_apps.sh#L36

From what I read, yes it is.

You probably right, I'll change that, moreover the status `inprogress` should remove level too.

**Comment by kay0u** [comment]
Should be ok now

File: change_level.py [hunk 1]
```diff
@@ -29,7 +29,11 @@
     if not 0 <= int(level) <= 10:
         print "Error: app level must be between 0 and 10, it's '%s'" % level
         sys.exit(1)
-
-    app_list[app_id]["level"] = int(level)
+        
+    if app_list[app_id]["state"] == "working":
+        app_list[app_id]["level"] = int(level)
+    else:
```
Comment (alexAubin) [inline change_level.py]: Hmm I think that might trigger an issue if the level key does not exist :P

```suggestion
    elif "level" in app_list[app_id]:
```

**Pull request merged**